app/components/utils/search-legend/template.hbs

```diff
@@ -0,0 +1,51 @@
+
+<table class="vl-data-table tr-hover">
+  <thead>
+    <tr>
+      <th class="vl-data-table-col-2 vl-data-table__header-title">
+        {{t "search-legend-header-example"}}
+      </th>
+      <th class="vl-data-table-col-9 vl-data-table__header-title">
+        {{t "search-legend-header-explanation"}}
+      </th>
+      <th class="vl-data-table-col-9 vl-data-table__header-title">
+        <button data-test-vl-modal-close
+                class="vl-button vl-button--link-muted vl-button--icon"
+          {{action "close"}}
+        >
+          <i
+                  class="vl-button__icon vl-vi vl-vi-close-light"
+                  aria-hidden="true"
+          ></i>
+          <span class="vl-u-visually-hidden">
+            {{t "close"}}
+          </span>
+        </button>
```

Comment:
Couple of things going on here:
* Funky indentation
* There seems to be a test tag from the modal that slipped into this button here
* The entire modal has `cursor: pointer` but the close button does not. That seems very counter-intuitive 😕

Reply:
I've applied some changes to the model. Introducing a new utillity class `vlc-u-cursor-default` to select the default pointer on the modal which results in the regular mouse. I've cursor: pointer only to the close button and removed the existing one completely from the form and its rows as the rows are non-clickable/draggable.